### PR TITLE
Add angellist

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Comment by Carlgo11** [comment]
Year 1999. We're getting closer to the 2k bug! 😃 
https://en.wikipedia.org/wiki/1999

**Comment by stephengroat** [comment]
pets.com or some other .com bubble era company?

**Comment by Carlgo11** [comment]
I'm just waiting to see what the 2000 issue/PR will be about 👀

File: _data/other.yml [hunk 1]
```diff
@@ -23,6 +23,13 @@ websites:
       tfa: No
       twitter: airbnb
 
+    - name: AngelList
+      url: https://angel.co
+      img: angellist.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+
```
Comment (Carlgo11) [inline _data/other.yml:32]: no doc link?

Comment (stephengroat) [reply _data/other.yml:32]: couldn't find anything in https://angel.co/help

**Pull request merged**